pkg/application/consul/options.go

```diff
@@ -0,0 +1,49 @@
+package consul
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+)
+
+type ConsulOptions struct {
+	application.ApplicationOptions
+	Namespace  string
```

Comment:
Namespace is available in ApplicationOptions so you can remove it here.

Reply:
Done